terraform_files/variables-libvirt.tf

```diff
@@ -134,3 +134,10 @@ variable "libvirt_master_disk" {
   description = "Disk size in bytes allocated to master"
   default     = "21474836480"
 }
+
+variable "running" {
+  type        = bool
+  description = "Start vms"
```

Comment:
Can you elaborate? Something like "Start Vms or just create its resources"

Reply:
it says , start vms or not :)

Reply:
Wasn't clear to me